app/static/responsive.css

```diff
@@ -35,6 +36,33 @@
     display: none;
   }
 
+  .show-menu {
+    display: block;
+    position: absolute;
+    top: 85px;
+    width: 335px;
+    z-index: auto;
+    height: 100%;
+    background-color: #6070ff;
+    margin-top: -85px;
+    text-align: start;
+    opacity: 0.8;
+  }
```

Comment:
**[Optional]**
- [ ] You forgot to add the backdrop filter to blur your menu. Kindly add it to make it similar to the Figma template 😊